src/Screens/Explore/components/DiveSite.tsx

```diff
@@ -79,12 +79,13 @@ const DiveSite: FunctionComponent<DiveSiteProps> = props => {
 
   if (props.onPressContainer && typeof props.onPressContainer === 'function') {
     return (
-      <TouchableWithoutFeedback
```

Comment:
The effects of this Touchable can get a little unsightly. Are you sure you want it?

Reply:
i tried it out and it seemed fine in the iOS simulator. TouchableHighlight seemed unsightly, is that what you're referring to?

Reply:
i also changed the opacity to 0.8, because the default was 0.2

Reply:
I think changing the opacity would do the trick. Approving. now.